package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@sfx/events",
+  "name": "@elements/events",
```

Comment:
Our package name is probably going to be `@groupby/elements-events`, but we can change that later when we actually release.